powerline/renderers/lemonbar.py

```diff
@@ -36,10 +37,26 @@ def hl(self, contents, fg=None, bg=None, attrs=None):
 		return text + contents + '%{F-B--u}'
 
 	def render(self, *args, **kwargs):
+		self.segment_info['output'] = kwargs.get('matcher_info')
```

Comment:
@S0lll0s Why? Copying of `self.segment_info` is performed by `Renderer.get_segment_info`. So does merging.